src/controllers/familyController.js

```diff
@@ -41,13 +42,38 @@ const createFamily = (req, res) => {
     });
 };
 
+const findFamily = key => new Promise((resolve, reject) => {
+  Family.findOne({ key }).then((foundFam) => {
+    if (!foundFam) {
+      reject(foundFam);
+    } else {
+      resolve(foundFam);
+    }
+  });
+});
+
+const join = (req, res) => {
+  const { key } = req.body;
+  findFamily(key).then((foundFam) => {
+    User.findById(req.userId, (findErr, foundUser) => {
+      foundUser.family_id = foundFam.id;
+      foundUser.save((err, updatedUser) => {
```

Comment:
updatedUser is never used, you don't need it here. This is the only error in eslint.